watchmaker/managers/base.py

```diff
@@ -191,40 +211,49 @@ def download_file(self, url, filename, sourceiss3bucket=False):
             except Exception as exc:
                 # TODO: Update `except` logic
                 logging.error('Unable to download file from web server.\n'
-                                  'url = {0}\n'
-                                  'filename = {1}\n'
-                                  'Exception: {2}'
-                                  .format(url, filename, exc))
+                              'url = {0}\n'
+                              'filename = {1}\n'
+                              'Exception: {2}'
+                              .format(url, filename, exc))
                 raise SystemError('Unable to download file from web server.\n'
                                   'url = {0}\n'
                                   'filename = {1}\n'
                                   'Exception: {2}'
                                   .format(url, filename, exc))
             logging.debug('Downloaded file from web server -- \n'
-                  '    url      = {0}\n'
-                  '    filename = {1}'.format(url, filename))
+                          '    url      = {0}\n'
+                          '    filename = {1}'.format(url, filename))
+
+    def call_process(self, cmd):
+        if not isinstance(cmd, list):
+            exceptionhandler('Command is not a list.\n{0}'.format(str(cmd)))
+        rsp = subprocess.call(cmd)
+
+        if rsp != 0:
```

Comment:
Ahh, ok. Do you want to rework call_process() or the exceptionhandler in another PR? We can open an issue to track the enhancement and mark this discussion resolved in reviewable.